src/vishwasCDec19/assignment3/Q14FindUniqueNo.java

```diff
@@ -0,0 +1,61 @@
+package vishwasCDec19.assignment3;
+import java.io.*; 
+public class Q14FindUniqueNo
+{
+	static void printUncommon(int arr1[],int arr2[], int n1, int n2) 
+	    { 
+	  
+	        int i = 0, j = 0, k = 0; 
+	        while (i < n1 && j < n2) 
```

Comment:
Very good Job @VishwasCDec19 
Could you please tell me this logic in class @VishwasCDec19